test/onnx/test_pytorch_onnx_no_runtime.py

```diff
@@ -427,7 +428,7 @@ def forward(self, x):
         onnx_model = export_to_onnx(
             MyClip(),
             torch.randn(3, 4, requires_grad=True),
-            custom_ops=[common_utils.custom_op("aten::clamp", bad_clamp, 9)],
+            custom_ops=[common_utils.custom_op("aten::clamp", bad_clamp, GLOBALS.export_onnx_opset_version)],
```

Comment:
This test was broken for a while. I wonder why CI didn't catch this.

Reply:
Found the current CI only has Caffe2 included version, so the tests with `skipIfCaffe2` is skipped. Those tests are all aten_fallback related.

Reply:
We need an issue for that. ONNX Runtime heavily depends on BUILD_CAFFE2=0 builds to properly work and PyTorch CI is not guarding those scenarios